app/views/presentations/edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Edit Presentation</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Back', presentations_path %>
```

Comment:
Kind of hesitant to assume that we know where the user is coming from. There is nothing to stop the user from just typing `presentations/1/edit` in their address bar. This back button would not take them to where they came from in that instance.